cpp/references/data_structures/list_digraph.hpp

```diff
@@ -0,0 +1,230 @@
+#ifndef LIST_DIGRAPH_HPP
+#define LIST_DIGRAPH_HPP
+
+#include <Godot.hpp>
+#include <Reference.hpp>
+
+#include <lemon/list_graph.h>
+#include <lemon/concepts/maps.h>
+
+namespace godot {
+
+#define _STR(m_x) #m_x
```

Comment:
I would actually suggest you move all of these macros into their own dedicated file, so that they may be used by other `godot-next` C++ code in the future. Just stick it in `cpp/err_macros.hpp` or something.